main.js

```diff
@@ -101,6 +100,13 @@ module.exports = function(options) {
 		});
 	}
 
+	if (packageJson.dependencies['next-metrics']) {
```

Comment:
can you just put the var packageJson a little higher up this file so that this won't syntax error if there's no package.json file?